### PR TITLE
Add more changes to the model so that it is consistent with paper

### PR DESCRIPTION
**Pull request by jtcramer**
* Move ReLU _after_ batch norm
* Use weight decay (assumed L2)
* Set learning rate to 1e-4
* Allow for volume gain less than 0dB, as paper seems like it should allow for that

### CODE REVIEW

**Comment by jtcramer** [comment]
If you specify an out file in the sbatch file, it will also save stdout from the script. But it would indeed be more robust to use logging rather than `print` statements.
